frontend/src/page/PetDetails.css

```diff
@@ -0,0 +1,18 @@
+.pet-details{
+    /* It is a empty css */
```

Comment:
why empty css? Maybe you could delete this.

Reply:
Hi., I had to do it, otherwise i become a "code smells " in Sonar Lint :-)